### ISSUE TITLE
Are the site and server working?

### ISSUE THREAD

**Issue by astrowonk** [immers-space/guppe#85]
I have not seen any boosted posts from the classical music guppe in the last two days on my instance. 

I tagged it today and I have no notification that it was boosted

Is anyone else having trouble?

**Comment by wmurphyrd** [comment]
I can see posts from within the last hour in that group. Please ask your server admin if a.gup.pe has been blocked or marked as unreachable 


![Screenshot](https://user-images.githubusercontent.com/10034859/218265942-321c64a3-9d67-4aa1-82f4-703edb87c739.jpg)

**Comment by vmstan** [comment]
So I'm the admin of the instance in question. It looks like you're sending empty user agent strings which attempt to post every few seconds-minutes which we blocked.

<img width="867" alt="CleanShot 2023-02-11 at 09 39 56@2x" src="https://user-images.githubusercontent.com/3002053/218267288-e7ff3394-7336-4c31-b084-731caa0e4281.png">

**Comment by vmstan** [comment]
If you fix your side to identify itself I’ll happily unblock it. I’m sure I’m not the only admin who saw that happening and thought something malicious was going on.

**Comment by wmurphyrd** [comment]
@vmstan Guppe only sends posts in direct response to requests from your users, so you're just interfering with their wishes.

Will open separate issue about filling in UA

**Comment by wmurphyrd** [comment]
@astrowonk you might have to consider moving servers if your admin won't stop blocking your desired use of Guppe. I will address their request when I'm.able, but we're still fundraising just to cover server costs

https://opencollective.com/guppe-groups

**Comment by vmstan** [comment]
I wasn’t going to comment on this since you’d closed it, and I’ve already discussed this privately with @astrowonk. 

There has been a huge uptick in the number of bots, scrapers, etc, trafficking Fediverse instances. No one is trying to “interfere with the wishes” of users here. I’m trying to balance the security of our users and manage our platform. Like you I’m a volunteer trying to balance my time spent in this space.

Since this particular traffic can be identified now, I’m going to work on exempting it from our firewall as soon as I’m able. I still think your software should properly identify itself so that you don’t run into this issue again.
